physionet-django/events/models.py

```diff
@@ -10,6 +11,11 @@
 from project.validators import validate_version, validate_slug
 
 
+class cohostStatus(IntEnum):
```

Comment:
Conventionally, classes in Python are PascalCase.

```suggestion
class CohostStatus(IntEnum):
```